engine/src/main/java/org/terasology/engine/rendering/logic/SkeletonRenderer.java

```diff
@@ -307,7 +307,7 @@ public void renderOpaque() {
                     boneTransform.mul(bone.getInverseBindMatrix());
                     boneTransforms[bone.getIndex()] = boneTransform.transpose();
                 } else {
-                    logger.warn("Unable to resolve bone \"{}\"", bone.getName());
+                    logger.atWarn().addArgument(() -> bone.getName()).log("Unable to resolve bone \"{}\"");
```

Comment:
Simple getter for member access.
```suggestion
                    logger.warn("Unable to resolve bone \"{}\"", bone.getName()); //NOPMD
```